frontend/src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
```

Comment:
Looks like not a comment anymore

Reply:
Sorry. My false. I have already fixed this issue.